webpack.config.js

```diff
@@ -1,11 +1,17 @@
 const TsconfigPathsPlugin = require("tsconfig-paths-webpack-plugin")
 const CopyWebPackPlugin = require("copy-webpack-plugin")
-// const WebpackShellPlugin = require('webpack-shell-plugin')
 
 const fileExtensions = [".ts", ".tsx", ".js", ".json"]
 
 module.exports = {
-    mode: "production",
+
+    mode: "development",
+    // mode: "production", // Uncomment me for more helpful error messages
```

Comment:
...Does `production` mode really provide more helpful error messages? @bovine3dom

Reply:
I actually have no idea - an exercise for the reader :)

When I initially wrote that comment I thought it was true.

Reply:
Heh, fair enough! :P